ocfweb/docs/docs/staff/backend/mesos.md

```diff
@@ -0,0 +1,202 @@
+[[!meta title="Mesos"]]
+
+At the OCF, we use [Mesos][mesos] and [Marathon][marathon] in order to deploy a
+number of important services. This page will try to document how all of these
+pieces fit together. The sub-pages are primarily troubleshooting steps and
+instructions for working with each individual component.
+
+
+## Mesos
+
+**[Apache Mesos][mesos]** is software originally developed at Berkeley which
+helps to schedule applications running on clusters of computers.
+
+From reading the Mesos website, it's difficult to tell exactly what Mesos does
+and does not do. The best way to think of Mesos is as a *scheduler*, which
+determines how to run a set of applications across a cluster of computers.
+
+
+### A Mesos cluster
+
+A Mesos cluster consists of a small number of Mesos masters, and a large number
+of Mesos agents (formerly called "slaves").
+
+At the OCF, we have three Mesos masters:
+
+* `mesos0` (`whirlwind`, running on `jaws`)
+* `mesos1` (`pileup`, running on `pandemic`)
+* `mesos2` (`monsoon`, running on `hal`)
+
+At any given time, a Mesos cluster has only one *leader*, which is always one
+of the Mesos masters. At any given time, only one of the masters is the leader,
+and the other two masters are effectively doing no real work (but keeping track
```

Comment:
so the other two masters are just there to ensure higher availability? (i.e. one takes over when leader dies)